util/keyfile/src/bin/convert_to_pubfile.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Processees pubkeys to file format understood by ingest add_user.
+//!
+//! This tool can be followed up by a call to add_user, e.g. with
+//! `cargo run fog-ingest-client -- --uri fog-ingest://fog-ingest.NETWORKNAME.mobilecoin.com:443 add-users --keys-path ./fog_keys`
+
+use mc_account_keys::PublicAddress;
+use mc_api::external;
+use mc_util_keyfile::write_pubfile;
+use protobuf::parse_from_bytes;
+use std::{convert::TryFrom, fs, path::PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Config {
+    /// Path to input pubfile dir
+    ///
+    /// Note: Pubfile format is expected to be a the hex-encoded protobuf bytes.
+    #[structopt(long)]
+    pub pubfile_in_dir: PathBuf,
```

Comment:
The `pubfile` prefix is confusing to me since it appears both on input and output but the type of files is going to be different.